config/redirects.yml

```diff
@@ -29,6 +26,19 @@ redirects:
 
   # International content
   "/iqts": "/non-uk-teachers/international-qualified-teacher-status"
+  "/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk": "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
+  "/international-teachers": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/international-students": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/international-candidates": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/explore-my-options/overseas-graduates": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/explore-my-options/overseas-applicants": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/how-to-apply/prepare/eu-nationals": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/overseas-applicants": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+  "/explore-my-options/return-to-teaching/return-to-teaching-in-england-from-overseas": "/non-uk-teachers/return-to-england-after-teaching-overseas"
+  "/node/3893": "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
```

Comment:
I think a couple of these are in here twice; like this one (also on line 445)

Reply:
Removed all the duplicates in bfb4a1f